.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+
+go:
+  - 1.4
+  - tip
+
+env:
+  - GOTAGS=restapi
+  #- GOTAGS=powershell
+
+install:
+  # trick to make go find packer-azure packages under MSOpenTech
+  - ln -s paulmey ../../MSOpenTech
+  - ls -al ../..
+  - go get -d -v -tags "$GOTAGS" ./...

```

Comment:
Hmmm, didn't even know about -t, Travis has -d -v in their defaults, but -t makes sense. I'll add it.